CMakeLists.txt

```diff
@@ -304,9 +315,5 @@ target_include_directories(${PROJECT_NAME}
 add_subdirectory(addkernels)
 
 # Installation
-install (TARGETS amd_rpp
-         LIBRARY DESTINATION /opt/rocm/rpp/lib
-)
-install (DIRECTORY include
-         DESTINATION /opt/rocm/rpp/
-)
+install (TARGETS amd_rpp LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
```

Comment:
@LakshmiKumar23 can you check if this works

Reply:
Yes rpp gets install at this path